scripts/run-sonata-sim.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${SONATA_SIMULATOR}" ] ; then
+	SONATA_SIMULATOR=/cheriot-tools/bin/sonata_simulator
+fi
```

Comment:
Purely stylistically, these can all be, for example,
```
: ${SONATA_SIMULATOR:=/cheriot-tools/bin/sonata_simulator}
```

Reply:
I've made this change now, thanks!